megago/megago.py

```diff
@@ -174,7 +176,7 @@ def get_default_go_dag():
     return GODag(GO_DAG_FILE_PATH, prt=open(os.devnull, 'w'))
 
 
-def run_comparison(go_list_1, go_list_2, go_dag=None):
+def run_comparison(go_list_1, go_list_2, go_dag=None, progress=None):
```

Comment:
could you add the progress parameter to the docstring? I do not understand what kind of parameter is expected. Is it a function?

Reply:
Done :)